nsqadmin/static/js/app_state.js

```diff
@@ -17,7 +17,11 @@ var AppState = Backbone.Model.extend({
         this.on('change:graph_interval', function(model, v) {
             localStorage.setItem('graph_interval', v);
         });
-        this.set('graph_interval', localStorage.getItem('graph_interval') || 'off');
+
+        var qp = _.object(_.compact(_.map(window.location.search.slice(1).split('&'),
+            function(item) { if (item) { return item.split('='); } })));
```

Comment:
I think that should work. Here it is without underscore (and in one loop):

```
var queryParams = window.location.search.slice(1).split('&').reduce(function(params, qStr) {
    if (qStr) {
        var pieces = qStr.split('=');
        params[pieces[0]] = pieces[1];
    }
    return params;
}, {});
```

Also, I think `array.map` is now supported by all major browsers (http://caniuse.com/#feat=es5), so there's also this:

```
var qp = _.object(_.compact(window.location.search.slice(1).split('&').map(function(item) {
    return item ? item.split('=') : null;
})));
```